jobs/JREGIONAL_REFL2TTEN

```diff
@@ -71,7 +71,7 @@ mkdir_vrfy -p ${WORKDIR}
 #-----------------------------------------------------------------------
 #
 $SCRIPTSDIR/exregional_refl2tten.sh \
-  CYCLE_DIR="${CYCLE_DIR}" WORKDIR="${WORKDIR}" || \
+  CYCLE_DIR="${CYCLE_DIR}" WORKDIR="${workdir}" || \
```

Comment:
same global uppercase local lowercase issue. yeah, it is a little bit difficult to follow this rule all the time.